database/java/2015/5211.yaml

```diff
@@ -0,0 +1,48 @@
+cve: 2015-5211
+title: "Spring Framework: RFD Download vulnerability"
+description: >
+    Under some situations, the Spring Framework is vulnerable to a Reflected File Download (RFD) attack.
+    The attack involves a malicious user crafting a URL with a batch script extension that results in the
+    response being downloaded rather than rendered and also includes some input reflected in the response.
+cvss_v2: 4.0
+references:
+    - http://pivotal.io/security/cve-2015-5211
+    - https://access.redhat.com/security/cve/cve-2015-5211
+
+    affected:
+    - groupId: "org.springframework"
+      artifactId: "spring-web"
+      version:
+        - "<=3.2.14,3.2"
+        - "<=4.2.1,4.2"
+        - "<=4.1.7,4.1"
+        - "<=4.1.7,4.0"
```

Comment:
I can do that, would I need to use the .RELEASE qualifier? (I was not sure if the `<=4.1.7,4.0` works but it does capture the Alerts intend of not giving 4.0 update instructions).